config/packages/nelmio_api_doc.yaml

```diff
@@ -823,6 +823,14 @@ nelmio_api_doc:
                         required: true
                         description: name
                         example: channel1
+            value:
```

Comment:
It's not true, 
value parameter have different structure due of attribute type is for

Reply:
Multiselect use array of options' ids as value